pom.xml

```diff
@@ -12,9 +12,10 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<openmrs.version>2.3.0</openmrs.version>
```

Comment:
Why the above version?

Reply:
@dkayiwa  i wanted to depend on a recent stable version of openmrs which is already released and in production

Reply:
Is that the most recent release?